client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx

```diff
@@ -38,7 +38,7 @@ const AddReviewDialog = ({ productId }) => {
     axios
       .post(`https://app-hrsei-api.herokuapp.com/api/fec2/hratx/reviews`, formDataStorage.data)
       .then(res => {
-      console.log(res)
+        handleSubmitReview(true);
```

Comment:
It seems to me that handleSubmitReview doesn't take any arguments, but I could be missing something...

Reply:
Good catch. I realized later that I don't need to pass ```true``` as a param and can just update state inside the event handler, but forgot to correct the function invocation here.